scripts/externalTests/download_benchmarks.py

```diff
@@ -141,32 +154,40 @@ def download_benchmarks(
 
     artifacts = circleci.artifacts(int(benchmark_collector_job['job_number']))
 
-    download_benchmark_artifact(artifacts, 'summarized-benchmarks', branch, actual_commit_hash, overwrite, silent)
-    download_benchmark_artifact(artifacts, 'all-benchmarks', branch, actual_commit_hash, overwrite, silent)
+    got_summary = download_benchmark_artifact(artifacts, 'summarized-benchmarks', branch, actual_commit_hash, overwrite, silent)
+    got_full = download_benchmark_artifact(artifacts, 'all-benchmarks', branch, actual_commit_hash, overwrite, silent)
+
+    return Status.OK if got_summary and got_full else Status.NO_BENCHMARK
 
 
 def main():
     try:
         options = process_commandline()
-        download_benchmarks(
+        return download_benchmarks(
             options.branch,
             options.pull_request_id,
             options.base_of_pr,
             options.ignore_commit_hash,
             options.overwrite,
             options.debug_requests,
-        )
-
-        return 0
+        ).value
+    except JobNotSuccessful as exception:
+        print(f"[ERROR] {exception}", file=sys.stderr)
+        if not exception.job_finished:
+            print("Please wait for the workflow to finish and try again.", file=sys.stderr)
+            return Status.PENDING.value
+        else:
+            print("Benchmarks from this run of the pipeline are not available.", file=sys.stderr)
+            return Status.NO_BENCHMARK.value
     except APIHelperError as exception:
         print(f"[ERROR] {exception}", file=sys.stderr)
-        return 1
+        return Status.ERROR.value
     except requests.exceptions.HTTPError as exception:
         print(f"[ERROR] {exception}", file=sys.stderr)
-        return 1
+        return Status.ERROR.value
     except RuntimeError as exception:
         print(f"[ERROR] {exception}", file=sys.stderr)
-        return 1
+        return Status.ERROR.value
 
 if __name__ == '__main__':
```

Comment:
Could you add an extra empty line before if statement to keep formatting consistent?

Reply:
Done.